test/integrations/routes/articles.test.js

```diff
@@ -104,6 +104,94 @@ describe('API endpoint: /api/articles (Routes)', () => {
           expect(res).to.have.status(STATUS.OK);
           expect(res.body).to.be.an('object');
           expect(res.body).to.haveOwnProperty('code').to.equal(STATUS.OK);
+          expect(res.body).to.haveOwnProperty('code').equal(STATUS.OK);
```

Comment:
#### Nitpick
- Very biased test 😉

Reply:
Thanks for pointing that out @aanchirinah, I have removed the unnecessary tests.